proto/pylons/pylons/accounts.proto

```diff
@@ -26,4 +26,12 @@ message ReferralKV {
 message RefereeSignup {
   string username = 1;
   string address = 2;
+}
+
+message KYCAddress {
```

Comment:
merge KYCAddress and KYCUsername

Reply:
Fix in https://github.com/Pylons-tech/pylons/pull/1018/commits/12f7d4c5080437ecede7d6cd979b06a3a88daa15